test/functionalTest/cases/1601_restrict_chars_in_ids/restrict_len_ids.test

```diff
@@ -51,7 +51,7 @@ echo "========================================="
 payload='{
   "id": "VERY_LONG",
   "type": "T1",
-  "España": {
+  "A": {
```

Comment:
After this PR, the check about forbidden chars in attributes was done before than the one in entity id length covered by this .test. Thus I have changed the attribute name to keep the .test result the same.

As long as the code change only involves a change in the order in which error conditions are raised and that doesn't have an impact in the response (appart from description, i.e. both are 400 Bad Request) I think this doesn't break backward compatibility.

However, I have preferred to describe the case in this comment in the case somebody have a different opinion :)